integration/generic-tests/ip_test.go

```diff
@@ -75,6 +75,32 @@ func TestIP(t *testing.T) {
 		cat /proc/net/route
 		#! grep -iq "$hex_destination" /proc/net/route || exit 1
 
+		# Add a neighbor (ARP entry) on eth0
+		cat /proc/net/arp
+		ip neigh add 192.168.1.2 lladdr 00:11:22:33:44:55 dev eth0 || exit 1
+		cat /proc/net/arp
```

Comment:
Is this for CI debugging/inspection, or can we remove that?

Reply:
Yes, this was for debugging purposes. I removed all instances.